atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/encoding/TimestampEncodingStrategy.java

```diff
@@ -27,7 +27,7 @@
  *
  */
 public interface TimestampEncodingStrategy<V> extends CellEncodingStrategy {
-    byte[] encodeCommitTimestampAsValue(long startTimestamp, V commitTimestamp);
+    byte[] encodeCommitStatusAsValue(long startTimestamp, V commitTimestamp);
```

Comment:
This was probably missed in an earlier PR, but `V commitTimestamp` should probably say `V commitStatus`.